src/modules/Host.rb

```diff
@@ -349,9 +376,7 @@ def SetModified
       nil
     end
 
-    publish :variable => :hosts, :type => "map <string, list <string>>"
     publish :variable => :modified, :type => "boolean"
-    publish :variable => :write_only, :type => "boolean"
```

Comment:
this removes also accessor, do you really check all SLE12 modules? It is API incompatible change, which is not nice in update branch